internal/featuresupport/support_levels_list.go

```diff
@@ -221,6 +241,10 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDLVM,
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
 			},
+			{
+				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDNUTANIXINTEGRATION,
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
```

Comment:
should be FeatureSupportLevelFeaturesItems0SupportLevelTechPreview  in 4.11

Reply:
Fixed

Reply:
Right, updated